tools/container/README.md

```diff
@@ -28,3 +28,17 @@ docker run \
     --env CAN_BUS0=<CAN_BUS0> \
     public.ecr.aws/aws-iot-fleetwise-edge/aws-iot-fleetwise-edge
 ```
+3. Alternatively credentials can be directly passed as environment variables to the container
+
+For example:
+```bash
+docker run \
+    -ti \
+    --network=host \
+    --env CERTIFICATE "-----BEGIN CERTIFICATE-----\nXXXXX\n-----END CERTIFICATE-----\n" \
```

Comment:
Needs `=` between variable and value, also next line.

Reply:
fixed